src/components/MultiLink/Legend.vue

```diff
@@ -272,6 +300,9 @@ export default Vue.extend({
           glyph: this.nestedVariables.glyph,
         };
         store.commit.setNestedVariables(updatedNestedVars);
+
+        // Render the scales
+        Vue.nextTick(() => this.renderScales(droppedElText));
```

Comment:
This feels a bit dangerous, though perhaps unavoidable. I think you could wrap the `renderScales()` method in a `watchEffect` if and when you refactor this to use composition API.

Reply:
See #186